Xamarin.Essentials/Background/Background.ios.service.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UIKit;
+
+namespace Xamarin.Essentials.Background
+{
+    public class BackgroundService
+    {
+        static nint taskId;
+        static bool isRunning;
+
+        /// <summary>
+        /// Start the execution of background service
+        /// </summary>
+        public static void Start()
+        {
+            if (isRunning)
+                return;
+
+            taskId = UIApplication.SharedApplication.BeginBackgroundTask("BackgroundTask", Stop);
+            Background.StartJobs();
+
+            isRunning = true;
+        }
+
+        public static void Stop()
```

Comment:
seems this method is not needed?

Reply:
Yes it's not implemented yet but I suppose we could use it to add features soon so I didn't went out of my way to get rid of it

Reply:
then let's remove the method, we'll add it when we need it

Reply:
but if Stop() is not called anywhere, even if you add something to it, it's unused, I'd say remove it until we need it